Meta/Lagom/Tools/CodeGenerators/LibWeb/BindingsGenerator/IDLGenerators.cpp

```diff
@@ -201,6 +202,9 @@ CppType idl_type_name_to_cpp_type(Type const& type, Interface const& interface)
     if (type.name() == "Function")
         return { .name = "JS::NonnullGCPtr<WebIDL::CallbackType>", .sequence_storage_type = SequenceStorageType::MarkedVector };
 
+    if (type.name() == "ImageBitmap")
```

Comment:
Since `ImageBitmap` is defined as a platform object, this shouldn't be reachable

Reply:
Right, I probably added that before I registered it in the platform object list